### PR TITLE
ADEN-13100 - unify - part 2 - proposal

### PR DESCRIPTION
**Pull request by adi-wesolowski-fandom**

### CODE REVIEW

**Review by rafalgiemza — APPROVED** [review]

File: src/platforms/news-and-ratings/metacritic-neutron/setup/context/slots/metacritic-neutron-slots-context.setup.ts [hunk 1]
```diff
@@ -74,197 +78,110 @@ export class MetacriticNeutronSlotsContextSetup implements DiProcess {
...
-			'mpu-top': {
-				defaultSizes: [[300, 250]],
-				targeting: {
-					pos_nr: 'top',
-				},
-			},
-			'mpu-middle': {
+			top_boxad: {
+				adProduct: 'top_boxad',
+				bidderAlias: 'mpu-top',
```
Comment (kwfandom) [inline src/platforms/news-and-ratings/metacritic-neutron/setup/context/slots/metacritic-neutron-slots-context.setup.ts:83]: idk if the bidder aliases should be also changed 🤔

File: src/platforms/news-and-ratings/metacritic-neutron/setup/context/slots/metacritic-neutron-slots-context.setup.ts [hunk 2]
```diff
@@ -74,197 +78,110 @@ export class MetacriticNeutronSlotsContextSetup implements DiProcess {
...
-					[5, 5],
+			incontent_boxad: {
+				adProduct: 'incontent_boxad',
+				bidderAlias: 'mpu-middle',
+				defaultSizes: [[300, 250]],
+				sizes: [
+					{
+						viewportSize: [840, 200],
+						sizes: [
+							[300, 250],
```
Comment (kwfandom) [inline src/platforms/news-and-ratings/metacritic-neutron/setup/context/slots/metacritic-neutron-slots-context.setup.ts:152]: 300, 600 is visible in the designs

Comment (adi-wesolowski-fandom) [reply src/platforms/news-and-ratings/metacritic-neutron/setup/context/slots/metacritic-neutron-slots-context.setup.ts:152]: so add it? ok

File: src/platforms/news-and-ratings/metacritic-neutron/setup/context/slots/metacritic-neutron-slots-context.setup.ts (hunk 1 above)
Comment (adi-wesolowski-fandom) [reply src/platforms/news-and-ratings/metacritic-neutron/setup/context/slots/metacritic-neutron-slots-context.setup.ts:83]: I dont change it :) it was using old nam, but with change of the name, I dont want change the name for bidders, thats why I put alias to keep old name (coz most propably, there is no new IDs set - I think it should be done in the new follow up task)

File: src/platforms/news-and-ratings/metacritic-neutron/setup/context/slots/metacritic-neutron-slots-context.setup.ts (hunk 2 above)
Comment (adi-wesolowski-fandom) [reply src/platforms/news-and-ratings/metacritic-neutron/setup/context/slots/metacritic-neutron-slots-context.setup.ts:152]: done

**Pull request closed without merging**